lib/serve.js

```diff
@@ -454,6 +454,20 @@ module.exports = bootstrap(function createServer(options, config, configFile) {
         addBeforeReadyTask('Schedule data cache background update', () => scheduleTasks.forEach(fn => fn()));
     }
 
+    addBeforeReadyTask('Schedule obsolete cache clean', () => {
+        setInterval(() => {
+            if (fs.existsSync(options.cache)) {
+                const cacheFiles = utils.getCacheFiles(options.cache);
+                for (const [, caches] of Object.entries(cacheFiles)) {
+                    const cachesSorted = caches.sort((a, b) => b.timestamp - a.timestamp);
+                    for (let i = 2; i < cachesSorted.length; i++) {
```

Comment:
Oh, looks like caches already sorted...
```js
for (const [, ...obsoleteCaches] of Object.values(cacheFiles)) {
    for (const { file } of obsoleteCaches) {
        fs.unlink(...)
    }
}
```

Reply:
Yeah, they're sorted. And yes I left two last cache files for purpose. In case we are updating cache we need to be sure that there's no reading/writing to the current cache and the previous version of it.